### DISCUSSION TITLE
If you do this on WLS, here's the fix

### DISCUSSION THREAD

**Discussion by Toasterme** [spotify2tidal/spotify_to_tidal#91] [Ideas]
If you do the transfert from WLS from windows 10-11, the browser must be linked to everything.  The easy quick way is to export the chrome browser from the OS with this command : 
export BROWSER="/mnt/c/Program Files/Google/Chrome/Application/chrome.exe"

Error was : 
![image](https://github.com/user-attachments/assets/ad48fe68-d423-4e7f-baa5-f763a84910a0)

Thanks for the creators, just transfered over 2500 songs! all for free.

**Comment by Simon-Str** [comment]
Thank you so much :) My terminal in WSL just did not show anything at all and I was wondering where its supposed to get the Tidal credentials from. I did not get an error either. But this was the fix for me!
